cxx-sensors/src/main/resources/cppcheck.xml

```diff
@@ -6345,7 +6345,1017 @@ The expression 'strcmp(x,"def") != 0' is suspicious. It overlaps 'strcmp(x,"abc"
     <remediationFunction>LINEAR</remediationFunction>
     <remediationFunctionGapMultiplier>1min</remediationFunctionGapMultiplier>
   </rule>
-  <!-- ########### add missing rules which seems not to be part of cppcheck errorlist ########### -->
+  <!-- ########### keys matching the schema <some-function-id>Called are not listed by the simple cppcheck -errorlist          ########### -->
+  <!-- ########### they become visible only if corresponding library was loaded, e.g. cppcheck -errorlist -library=posix.cfg   ########### -->
+  <!-- ########### see generate_cppcheck_resources.sh for more details                                                         ########### -->
+  <rule>
```

Comment:
similar to all existing obsolete/ not thread-safe functions the new ones have

```XML
<severity>MINOR</severity>
<type>CODE_SMELL</type>
```

Not sure it's really that harmless, but at least it's consistent